src/Ledger/PParams.lagda

```diff
@@ -112,6 +113,156 @@ paramsWellFormed pp =
                    ∷ govActionLifetime ∷ govActionDeposit ∷ drepDeposit ∷ [] )
   where open PParams pp
 \end{code}
+\begin{code}[hide]
+record PParamsUpdate : Set where
```

Comment:
How about putting this type & everything that relates to it in a module? Then you can leave `UpdateT` renamed to `PParamsUpdate` below, and don't have to change all the other files.

This should also fix the additional `⦃ _ : DecEq UpdateT ⦄` arguments you needed to introduce, since there'a a `⦃ DecEq-UpdT ⦄ : DecEq PParamsUpdate` field, but removing the renaming makes this the wrong instance.